src/Pages/SignIn/SignIn.scss

```
@@ -0,0 +1,193 @@
+.SignIn {
+  width: 580px;
+  margin: 0 auto;
+  background-color: white;
+  font-family: notokr-medium;
+
+  header {
+    display: flex;
+    justify-content: center;
+  }
+
+  main {
+    p {
+      text-align: center;
+      font-size: 20px;
+      em {
+        color: #009ee3;
+        font-style: normal;
+      }
+    }
+
+    div {
+      .memberSignIn {
+        border: 1px solid #d9d9d9;
```

```suggestion
      .memberSignIn {
        border: 1px solid #d9d9d9;
```
div 태그로 nesting하지 않아도 하위에 있는 selector들이 충분히 동작할 것 같습니다. 삭제해주세요